index.php

```diff
@@ -8,52 +8,133 @@
     <title>Ballers: Become A Landlord [COMING SOON]</title>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <link rel="stylesheet" href="assets/css/styles.css">
+
   </head>
   <body>
     <div class="container">
-        <div class="form-container col-md-8 col-sm-12">
-          <header class="text-center">
-            <img src="assets/img/logo.png" alt="Ballers Logo" class="logo">
-          </header>
 
-          <section>
-            <h1 class="title text-center">Become a Landlord</h1>
-            <p class="text-center text-muted">Join a community of insightful minds where you can own a home with ease.</p>
-          </section>
+        <!-- Header -->
+        <header>
+          <nav class="navbar">
+              <a class="navbar-brand" href="#">
+                <img src="./assets/img/logo.png" class="logo" alt="ballers_logo">
+              </a>
+              <h5 class="text-right nav-text">Strategically build your wealth today.</h5>
+          </nav>
+        </header>
+        <!-- /.Header -->
+
+        <!-- Holding Page -->
+        <section class="holding">
+            <h4 class="ball-full-text">BECOME A LANDLORD</h4>
+            <br><br>
+            <h1 class="coming-soon">COMING SOON</h1>
+            <p class="drop-text">Drop your details below to be contacted <br> once the Ballers platform is live</p>
+        </section>
+        <!-- /.Holding Page -->
+
+        <!-- Notification Form -->
+        <aside class="contact-form col-md-10 ">
+          <form data-toggle="validator" method="post" action="#">
+            <div class="row">
+              <div class="col-md-9">
+                <div class="row">
 
-          <form data-toggle="validator" method="post" action="#" class="mt-5 pt-3">
-            <p class="text-left">Pls fill out your details below to be notified when we launch.</p>
+                  <?php require_once('inc/sendmail.php'); ?>
 
-            <?php require_once('inc/sendmail.php'); ?>
+                  <div class="form-group col-md-4">
```

Comment:
Can we remove all the placeholder element and introduce the floating label effect introduced by the designer. I mean the entire block after the required statement should look like this
```suggestion
              <div class="form-group col-md-4">
                    <input type="text" class="form-control" name="full_name" required>
                    <label for="full_name">Full name</label>
                    <small class="help-block with-errors text-danger"></small>
                  </div>
```

This should be repeated for the email and phone fields.

Reply:
After updating the label fields, adding this to your CSS should create the label effect
```css
form {
  position: relative;
}

label {
  position: absolute;
  color: #c0ccda;
  top: 0.6rem;
  left: 1rem;
  font-size: 1rem;
  cursor: text;
  transition: top 0.2s ease-in-out, font-size 0.2s ease-in-out;
}

input:focus {
  outline: none;
}

input:focus + label,
input:valid + label,
.has-error input:invalid + label {
  top: -0.9rem;
  font-size: 0.8rem;
}

.has-error input:invalid {
  border-color: #d93025;
}

.has-error input:invalid + label {
  color: #d93025;
}

.has-error label {
  color: #d93025;
}

.with-errors ul {
  margin-top: 0.5rem;
}
```